code/__HELPERS/_macros.dm

```diff
@@ -131,6 +131,10 @@
 
 #define isdeathsquad(H) (H.mind in ticker.mode.deathsquads)
 
+
+#define ishulk(H) (M_HULK in H.mutations)
```

Comment:
I'm pretty sure this is fucked up and doesn't actually do what you think it does, see #7092